scene/gui/rich_text_label.cpp

```diff
@@ -5106,16 +5139,17 @@ bool RichTextLabel::search(const String &p_string, bool p_from_selection, bool p
 		if (selection.from_frame->parent != nullptr && selection.from_frame->parent->type == ITEM_TABLE) {
 			// Find last search result in table
 			ItemTable *parent_table = static_cast<ItemTable *>(selection.from_frame->parent);
-			List<Item *>::Element *parent_element = p_search_previous ? parent_table->subitems.back() : parent_table->subitems.front();
+			List<Item *> frames = _get_table_frames(parent_table);
+			List<Item *>::Element *parent_element = p_search_previous ? frames.back() : frames.front();
 
 			while (parent_element->get() != selection.from_frame) {
 				parent_element = p_search_previous ? parent_element->prev() : parent_element->next();
 				ERR_FAIL_NULL_V(parent_element, false);
 			}
 
 			// Search remainder of table
-			if (!(p_search_previous && parent_element == parent_table->subitems.front()) &&
-					parent_element != parent_table->subitems.back()) {
+			if (!(p_search_previous && parent_element == frames.front()) &&
+					parent_element != frames.back()) {
```

Comment:
```suggestion
			if (!(p_search_previous && parent_element == frames.front()) && parent_element != frames.back()) {
```
Not needed now with narrower line